roles/sap_ha_pacemaker_cluster/defaults/main.yml

```diff
@@ -135,6 +135,20 @@ sap_ha_pacemaker_cluster_hana_filesystem_resource_name: >-
 sap_ha_pacemaker_cluster_hana_filesystem_resource_clone_name: >-
   cln_SAPHanaFil_{{ sap_ha_pacemaker_cluster_hana_sid }}_HDB{{ sap_ha_pacemaker_cluster_hana_instance_nr }}
 
+# SAP HANA - Constraint names
+sap_ha_pacemaker_cluster_hana_order_topology_hana_name: >-
+  ord_saphana_saphanatop_{{ sap_ha_pacemaker_cluster_hana_sid }}_HDB{{ sap_ha_pacemaker_cluster_hana_instance_nr }}
+
+sap_ha_pacemaker_cluster_hana_colocation_hana_vip_primary_name: >-
+  col_saphana_vip_{{ sap_ha_pacemaker_cluster_hana_sid }}_HDB{{ sap_ha_pacemaker_cluster_hana_instance_nr }}_primary
+sap_ha_pacemaker_cluster_hana_colocation_hana_vip_secondary_name: >-
+  col_saphana_vip_{{ sap_ha_pacemaker_cluster_hana_sid }}_HDB{{ sap_ha_pacemaker_cluster_hana_instance_nr }}_primary
```

Comment:
I assume this should have the `_readonly` suffix?

Reply:
@ja9fuchs Thank you, I have it fixed locally and I will push it along with crmsh post steps that I am working on.